misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -26,31 +26,62 @@ interface HttpCall {
   fun setResponseHeader(name: String, value: String)
   fun addResponseHeaders(headers: Headers)
 
+  /**
+   * Call this before the response body is written to make sure it is encoded in a way that'll
+   * permit trailers to be sent. This will do chunked encoding for HTTP/1. For HTTP/2 trailers are
+   * always permitted. It is an error to call this for web socket calls.
+   */
+  fun requireTrailers()
+
+  /**
+   * Add a trailer. This requires that [requireTrailers] was called before the response body is
+   * written.
+   */
+  fun setResponseTrailer(name: String, value: String)
```

Comment:
Let's rename it to `addResponseTrailer`? Otherwise feels like it's overriding

Reply:
Yes! Lemme fix in follow-up